.circleci/config.yml

```diff
@@ -156,6 +156,29 @@ jobs:
       - store_artifacts:
           path: integration-tests/screenshots
 
+  upload_test_data_to_s3:
+    executor:
+      name: hmpps/default_small
+    steps:
+      - checkout
+      - hmpps/k8s_setup
+      - hmpps/install_aws_cli
+      - run:
+          name: Upload test data to S3 bucket
+          command: |
+            S3_BUCKET_NAME=$(kubectl --namespace="${KUBE_ENV_NAMESPACE}" get secret analytical-platform-s3-bucket-output -o go-template='{{.data.bucket_name|base64decode}}')
+            export AWS_ACCESS_KEY_ID=$(kubectl --namespace="${KUBE_ENV_NAMESPACE}" get secret analytical-platform-s3-bucket-output -o go-template='{{.data.access_key_id|base64decode}}')
+            export AWS_SECRET_ACCESS_KEY=$(kubectl --namespace="${KUBE_ENV_NAMESPACE}" get secret analytical-platform-s3-bucket-output -o go-template='{{.data.secret_access_key|base64decode}}')
+            aws s3 sync server/testData/s3Bucket s3://${S3_BUCKET_NAME} || export UPLOAD_FAILED=1
+            unset AWS_ACCESS_KEY_ID
+            unset AWS_SECRET_ACCESS_KEY
```

Comment:
unset secrets just in case before exiting, whether or not `aws s3 sync` was successful